docs/_static/css/custom.css

```diff
@@ -0,0 +1,28 @@
+@import url('https://fonts.googleapis.com/css2?family=Ubuntu:ital@0;1&display=swap');
```

Comment:
Is this something I should be incorporating back into the base?

Reply:
I think so! I'm making a list though, you don't have to worry about it (yet)